cross/vo-amrwbenc/Makefile

```diff
@@ -0,0 +1,31 @@
+PKG_NAME = vo-amrwbenc
+PKG_VERS = 0.1.3
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://sourceforge.net/projects/opencore-amr/files/$(PKG_NAME)
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS =
+
+HOMEPAGE = https://github.com/msstorsjo/vo-amrwbenc/
+COMMENT  = VisualOn AMR-WB encoder library
+LICENSE  = Apache 2
+
+GNU_CONFIGURE = 1
+
+PRE_CONFIGURE_TARGET = myPreConfigure
+
+ifeq ($(findstring $(ARCH), $(ARM5_ARCHES)),$(ARCH))
```

Comment:
See my earlier comment on this: please verify that if the arg is needed, that it's actually set.